tests/integration/test_executable_table_function/test.py

```diff
@@ -13,6 +13,12 @@
 node = cluster.add_instance('node', stay_alive=True, main_configs=[])
 
 
+# Something like https://reviews.llvm.org/D33325
```

Comment:
But this is about regular fork (and AFAICS now *San is fork safe), vfork should not have such issues. Are there any details?

P.S. I've tried to run this tests with MSan binary and cannot reproduce (but I guess it is not permanent), also I've looked at checks database, but all reports are outdated.